src/components/SlateEditor/plugins/file/DndFileList.tsx

```diff
@@ -0,0 +1,147 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+// eslint-disable-next-line lodash/import-scope
+import { DebouncedFunc } from 'lodash';
+import debounce from 'lodash/debounce';
```

Comment:
No berre gjetter eg, men kan dette spesifiseres slik?
```suggestion
import debounce, { DebouncedFunc } from 'lodash/debounce';
```

Reply:
Mener å huske nei, faktisk

Reply:
Nei :/ `DebounceFunc` finnes ikke i lodash/debounce

Reply:
Rart av lodash, men ingenting å gjøre noe med gitt.